docs/guides/1.2.0-upgrade-guide.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: "couchbase-capella"
+page_title: "Couchbase Capella Provider 1.2.0: Upgrade and Information Guide"
+sidebar_current: "docs-couchbase-capella-guides-120-upgrade-guide"
+description: |-
+Couchbase Capella Provider 1.2.0: Upgrade and Information Guide
+---
+
+# Couchbase Capella Provider 1.2.0: Upgrade and Information Guide
+
+Here is a list of what's new in 1.2.0
+
+## New Features
```

Comment:
switch the order of features below:

- Enable audit logging for clusters and manage audit log settings
- Export Audit logs
- Get events